sdk_container/src/third_party/coreos-overlay/coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -81,6 +81,7 @@ RDEPEND="${RDEPEND}
 
 # sys-devel/gettext: it embeds 'envsubst' binary which is useful for simple file templating.
 RDEPEND="${RDEPEND}
+	app-admin/azure-nvme-utils
```

Comment:
Maybe a changelog entry would be good under `changes/` to see when this is available in Stable and also understand how it's used/what the effects are.

Reply:
added an entry